configure.ac

```diff
@@ -852,6 +852,8 @@ fi
 if test "x$AOT_BUILD_FLAGS" != "x"; then :
    AC_SUBST(AOT_BUILD_FLAGS)
    AC_SUBST(AOT_RUN_FLAGS)
+   # For llvmonlycheck + fullaotcheck
+   AC_SUBST(INVARIANT_AOT_OPTIONS)
 fi
```

Comment:
It would be easier to put these flags directly into the Makefile, unless we want to share it between makefiles.